upgrade/1.2/scripts/common/ncn-rebuild-common.sh

```diff
@@ -36,8 +36,8 @@ state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
 if [[ $state_recorded == "0" && $2 != "--rebuild" ]]; then
     echo "====> ${state_name} ..."
 
-    if ! cray bss bootparameters list --hosts $target_ncn --format json | jq '.[] |."cloud-init"."user-data".ntp' | grep -q '/etc/chrony.d/cray.conf'; then
-        echo "$(target_ncn) is missing NTP data in BSS. Please see the procedure which can be found in the 'Known Issues and Bugs' section titled 'Fix BSS Metadata' on the 'Configure NTP on NCNs' page of the CSM documentation."
+    if ! cray bss bootparameters list --hosts $TARGET_XNAME --format json | jq '.[] |."cloud-init"."user-data".ntp' | grep -q '/etc/chrony.d/cray.conf'; then
+        echo "${target_ncn} is missing NTP data in BSS. Please see the procedure which can be found in the 'Known Issues and Bugs' section titled 'Fix BSS Metadata' on the 'Configure NTP on NCNs' page of the CSM documentation."
```

Comment:
I would shorten 
`Please see the procedure which can be found in the`
to
`Please see the procedure in the`